### PR TITLE
feat(action): fix working directory for dependency installation

### PR DESCRIPTION
**Pull request by marns93**
Fixes: https://github.com/moneymeets/action-beanstalk-deploy/issues/21

**HINT** Can be merged after https://github.com/moneymeets/action-setup-python-poetry/pull/9 is merged. And we need to update 
`uses: moneymeets/action-setup-python-poetry@feature/working-directory` 
to
`uses: moneymeets/action-setup-python-poetry@master`

### CODE REVIEW

**Review by mm-ivanovig — APPROVED** [review]

**Review by zyv — APPROVED** [review]
👍

File: action.yml [hunk 1]
```diff
@@ -28,12 +28,12 @@ runs:
   using: "composite"
   steps:
     - name: Setup Python + Poetry
-      uses: moneymeets/action-setup-python-poetry@master
+      uses: moneymeets/action-setup-python-poetry@feature/working-directory
```
Comment (felix11h) [inline action.yml:31]:
```suggestion
      uses: moneymeets/action-setup-python-poetry@master
```

**Review by felix11h — APPROVED** [review]

**Comment by felix11h** [comment]
I have added a fixup to update the action to use our usual naming scheme. I haven't tested that this resolves the problem described in https://github.com/moneymeets/action-beanstalk-deploy/issues/21, but from all I understand the approach makes sense to me and I would merge this pull request tomorrow.

**Comment by zyv** [comment]
Thank you!

**Pull request merged**